### DISCUSSION TITLE
2.6.5

### DISCUSSION THREAD

**Discussion by arianneorpilla** [arianneorpilla/jidoujisho#199] [Announcements]
This is a hotfix release for  [**2.6**](https://github.com/lrorpilla/jidoujisho/releases/tag/2.6.0) which includes **breaking changes for existing users**. Please refer to the [**releases page**](https://github.com/lrorpilla/jidoujisho/releases) for a full timeline of changes. 

**System Requirements**: This application has been tested to work on a variety of devices and emulators running Android 7 to 13. Please note that a significant amount of internal storage space may be taken up by the dictionary database for indexing in order to optimize query speed and performance. Try to give the application around 2-4GB of storage space. Make sure your Android WebView is updated before use. This will resolve common issues that may occur with ッツ Ebook Reader or Mokuro.

## ⚠️ Breaking Change
>There was a very poor decision with the original `jidoujisho Yuuna` template to not include the `image` and `sound` HTML tags in each card. This version will now make exporting with HTML tags on by default, with an updated default note type.
>
>If you `Clear Media`, all your media will not be detected by AnkiDroid, and you may end up deleting all your images and audio exported for the app. **Do not attempt this if you want to apply the fix to your note type. If you do so, you WILL lose media.** 
>
>You may continue use of the app if you never use `Clear Media`. However, if you wish to fix this issue, the steps are as follows. **Before doing anything, backup your** `collection.media` **folder and export your decks as** `.apkg`. Make sure you back up **both**.
>
> **Do not sync to AnkiWeb until you can confirm everything is working after the migration.**
>
>- Import your backup `.apkg` to Anki for desktop.
>- Select all `jidoujisho Yuuna` cards.
>- Find and replace `Image` field, find `(.+)` and replace with `<img src="${1}"></img>`. Use regular expressions.
>- Find and replace `Term Audio` and `Sentence Audio` field, find `(.+)` and `[sound:${1}]`. Use regular expressions.
>- Remove tags for `img` and `sound` from the back of the `jidoujisho Yuuna` note type.
>- Export from Anki and import the `.apkg` to AnkiDroid.
>- Using jidoujisho, go to the `Standard` profile and toggle on `Include image/audio HTML tags on export`.  
>
> If you have been using the `jidoujisho Yuuna` template, your cards will need the following changes above. However, so long as you do not use `Clear Media`, you will not require this migration. **Exporting with HTML tags will be on by default with every future iteration of the app moving forward.** Toggle this off with each fresh install of the app if you don't desire to migrate. If you don't have a `jidoujisho Yuuna` note type, you will not have HTML tags included with your template, and you will not require the changes above.  
>
> Again, so long as you do not use `Clear Media`, you may simply just update and continue to use the app. It is only when you will require its use that you will need to perform the fix above, as well as toggling off the option to export with HTML tags if fresh installing the app.
>
>**Due to this drastic change, this notice will be included moving forward with any future [2.6](https://github.com/lrorpilla/jidoujisho/releases/tag/2.6.0) releases**. I apologise for what this change may cause, as I understand the hard work you have put to make and collect your cards, and I want to be transparent of this issue before this causes anyone any trouble.

# 2.6.5
- Fixed an issue where a video could not be exited until it has loaded.
- Fixed an issue where the image carousel in the Card Creator would not always show the first image again upon new image results.
- Adjusted back the threshold required with the vertical swipe required to open the Transcript in the Player.

# 📲 Which file should I download?
As a rule of thumb, `arm64` should work for most modern mobile devices. `armeabi` should work on older devices. Generally, desktops and emulators should use `x86_64`.

Depending on which device you are using, download the appropriate `APK` by expanding the `Assets` drop-down below and install on your mobile phone by launching the `APK` with your file explorer.

<hr /><em>This discussion was created from the release <a href='https://github.com/lrorpilla/jidoujisho/releases/tag/2.6.5'>2.6.5</a>.</em>

**Comment by dominiieplay** [comment]
Hi, the site https://leermangamokureado.duckdns.org/ is not working anymore, is there another site to read maga in mokoro format?

**Reply by arianneorpilla** [reply]
Hi, please note the following link is not associated with this project.
https://mokuro.moe/manga/Manga/

**Reply by dominiieplay** [reply]
thank you sir!
